Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -163,6 +164,58 @@ const Info<std::string> MAIN_RESOURCEPACK_PATH{{System::Main, "General", "Resour
 const Info<std::string> MAIN_FS_PATH{{System::Main, "General", "NANDRootPath"}, ""};
 const Info<std::string> MAIN_SD_PATH{{System::Main, "General", "WiiSDCardPath"}, ""};
 const Info<std::string> MAIN_WFS_PATH{{System::Main, "General", "WFSPath"}, ""};
+const Info<bool> MAIN_SHOW_LAG{{System::Main, "General", "ShowLag"}, false};
+const Info<bool> MAIN_SHOW_FRAME_COUNT{{System::Main, "General", "ShowFrameCount"}, false};
+const Info<std::string> MAIN_WIRELESS_MAC{{System::Main, "General", "WirelessMac"}, ""};
+const Info<std::string> MAIN_GDB_SOCKET{{System::Main, "General", "GDBSocket"}, ""};
+const Info<int> MAIN_GDB_PORT{{System::Main, "General", "GDBPort"}, -1};
+const Info<int> MAIN_ISO_PATH_COUNT{{System::Main, "General", "ISOPaths"}, 0};
+
+std::vector<std::string> GetIsoPaths()
+{
+  size_t count = MathUtil::SaturatingCast<size_t>(Config::Get(Config::MAIN_ISO_PATH_COUNT));
+  std::vector<std::string> paths;
+  paths.reserve(count);
+  for (size_t i = 0; i < count; ++i)
+  {
+    Config::Info<std::string> iso_path_info{
+        {Config::System::Main, "General", fmt::format("ISOPath{}", i)}, ""};
+    std::string iso_path = Config::Get(iso_path_info);
+    if (!iso_path.empty())
+      paths.emplace_back(std::move(iso_path));
+  }
+  return paths;
+}
+
+void SetIsoPaths(const std::vector<std::string>& paths)
+{
+  size_t old_size = MathUtil::SaturatingCast<size_t>(Config::Get(Config::MAIN_ISO_PATH_COUNT));
+  size_t new_size = paths.size();
+
+  size_t current_path_idx = 0;
+  for (const std::string& p : paths)
+  {
+    if (p.empty())
+    {
+      --new_size;
+      continue;
+    }
+
+    Config::Info<std::string> iso_path_info{
+        {Config::System::Main, "General", fmt::format("ISOPath{}", current_path_idx)}, ""};
+    Config::SetBase(iso_path_info, p);
+    ++current_path_idx;
+  }
+
+  for (size_t i = current_path_idx; i < old_size; ++i)
+  {
+    Config::Info<std::string> iso_path{
+        {Config::System::Main, "General", fmt::format("ISOPath{}", i)}, ""};
+    Config::SetBase(iso_path, "");  // TODO: This actually needs a Config::Erase().
```

Comment:
We don't seem to have a way to actually remove a key from the new config system, so I've left this like this for now.

Reply:
You could call DeleteKey() on the relevant layer but yeah I think this is fine for now.